apps/ichat-gw/ichat-gw.config

```diff
@@ -143,4 +143,9 @@ TLSPrivateKey =
 # Pass phrase of private key file (can be left blank)
 TLSPrivateKeyPassPhrase =
 
+########################################################
+# iChat Jabber Connector Settings
+########################################################
 
+# Path to iChat jabber connector (ichat-gw-jc) executable
+IchatJabberConnectorPath = "ichat-gw-jc"
```

Comment:
Does it need the double quotes?

Add another line like this example, commented out:

```
# For Windows:
#IchatJabberConnectorPath = ichat-gw-jc.exe